php/class-customize-concurrency.php

```diff
@@ -14,4 +14,204 @@
  */
 class Customize_Concurrency {
 
-}
+	/**
+	 * Post type that stores settings changes until session ends.
+	 *
+	 * Limited by WP to 20 characters
+	 *
+	 * @var string $POST_TYPE
+	 */
+	const POST_TYPE = 'custom_saved_setting';
```

Comment:
I like `customize_setting`, too but it could easily conflict with future development. Could we do `customize_c9y`, or some variation of that?

Whatever we choose will only be seen on this one line of code and nowhere in the UI, so we could also use a random string. :)